src/components/Accordion/FormAccordion.tsx

```diff
@@ -58,21 +59,40 @@ const HeaderWrapper = styled.div`
   }
 `;
 
+const StyledAccordionContent = styled(AccordionContent)`
+  &[data-frontpage='true'] {
```

Comment:
Kanskje det er fordelaktig for oss å holde oss til `data-wide`? Da knytter vi oss ikke så tett til tanken om at det kun er forsideartikler som er brede.

Reply:
Her er det mer at det er `frontpage-articles` som skal ha styling blå i editor samt. att det er kun de som skal bli wide. Kan eventuelt flytte `data-wide` ut men det vil ikke gi noe verdi da contentet under ikke vokser.